docs/usage.md

```diff
@@ -32,12 +32,24 @@ cond_4,assembly,testdata/test_minigut.contigs.fa.gz,A*01:01,testdata/test_minigu
 | ----------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `condition`       | The condition name for this entry. Conditions have to be unique and describe a combination of a microbiome, alleles and weights.                                                                           |
 | `type`            | Input type, can be one of "assembly", "bins" or "taxa".                                                                                                                                                    |
-| `microbiome_path` | Full path to microbiome file, the format of which can vary with type: fasta, folder, compressed folder or list of taxon ids.                                                                               |
+| `microbiome_path` | Full path to microbiome file, the format of which can vary with type: fasta, folder, compressed folder or tsv file for taxon ids (taxon_id ["\\t" assembly_ids "\\t" abundance]).                          |
 | `alleles`         | List of alleles to predict epitopes for.                                                                                                                                                                   |
 | `weights_path`    | Full path to a tab-separated file contataining weights. Currently allowed are contig weights for input types assembly and bins. Please use "contig_name" or "bin_basename" and "weight" as column headers. |
 
 An [example samplesheet](../assets/samplesheet.csv) has been provided with the pipeline.
 
+### Input type taxa
+
+The input type taxa allows the user to specify a taxid on strain level, for which all proteins are downloaded. The microbiome path corresponds to a tsv file containing one column for taxon_id and optionally specific assembly_ids and/or the abundance of a specific strain (see below). If only the taxid(s) (and optionally abundance) is provided, the pipeline will automatically download the largest assembly of the given taxid(s). If a specific assembly_id is provided it will download proteins of the given assembly_id. The input allows for a mixed assignment of specific assembly_ids and unspecific taxon_ids, but this is only recommended for specific use cases.
+
+As the pipeline is only generating reproducible results for this type of input if a specific assembly_id is chosen, it is highly recommended to use this option.
```

Comment:
Describe maybe how the pipeline can be run to reproduce results from previous runs which had no assembly_ids specified.